composer.json

```diff
@@ -68,6 +66,8 @@
         "phpunit/phpunit": "^9.5",
         "psalm/plugin-phpunit": "^0.16",
         "psalm/plugin-symfony": "^3.0",
+        "sonata-project/cache": "^2.0",
+        "sonata-project/cache-bundle": "^3.3",
```

Comment:
We should add this same range on the conflict section.

Reply:
added the lower versions to conflict